### PR TITLE
fix header and backgrounds

### PR DESCRIPTION
**Pull request by nester7x**
I've fixed header and deleted some backgrounds

### CODE REVIEW

File: css/components/header.css [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+    align-items: center;
 }
 
 .header__nav{
     display: flex;
     justify-content: space-between;
     flex-grow: 1;
-    min-width: 700px;
-    padding: 0 20px;
+    min-width: 600px;
```
Comment (SerhiienkoRuslan) [inline css/components/header.css:16]: max-width

File: css/components/header.css [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+    padding: 0 50px;
```
Comment (SerhiienkoRuslan) [inline css/components/header.css:17]: то много

File: css/components/section_1.css [hunk 3]
```diff
@@ -12,3 +15,25 @@
...
+h1 {
+    color: #ffffff;
+    font-size: 24px;
+}
+
+.learnMore {
+    text-decoration: none;
+    color: #ffffff;
+    font-size: 18px;
+    border-bottom: 1px solid #ffffff;
```
Comment (SerhiienkoRuslan) [inline css/components/section_1.css:32]: text-decoration: underline

File: css/components/section_1.css [hunk 4]
```diff
@@ -12,3 +15,25 @@
...
+}
+
+.learnMore:hover {
+    color: chocolate;
+    transition: .3s;
+    border-bottom: 1px solid chocolate;
```
Comment (SerhiienkoRuslan) [inline css/components/section_1.css:38]: text-decoration: none

File: index.html [hunk 5]
```diff
@@ -23,14 +23,14 @@
...
-      <div class="header__buttons">
-        <button class="btn">Journal</button>
-        <button class="btn">Search</button>
-        <button class="btn">Login</button>
-        <button class="btn">
+      <div class="header__desk">
+        <a class="btn">Journal</a>
+        <a class="btn">Search</a>
+        <a class="btn">Login</a>
+        <a class="btn">
```
Comment (SerhiienkoRuslan) [inline index.html:30]: href <----- обязательный аттрибут

**Comment by nester7x** [comment]
I've fixed .learnMore and .header__nav. I think min-width 700px is a lot of length, it doesn't fit in a container.

**Review by SerhiienkoRuslan — APPROVED** [review]

**Pull request merged**